NEWS.md

```diff
@@ -1,25 +1,30 @@
-# rextendr (development version)
+# rextendr 0.3.0
```

Comment:
I think this is done by `usethis::use_version()`, so no needs to do in this PR.

Reply:
Yes. I think it's fine to do it either in this PR or in another PR. I mean, as rextendr's repo doesn't allow you to push the changes to master directly, you'll need to do it in pull request anyway.

Reply:
Yep, I postponed it since it should happen at a later phase of submission (see #277).